example/nf_channels_params_template.json

```diff
@@ -0,0 +1,23 @@
+{
+"container_fastqc" : "/path/to/singularity/image/fastqc.sif",
+"container_multiqc" : "/path/to/singularity/image/multiqc.sif",
+"container_trimmomatic" : "/path/to/singularity/image/trimmomatic.sif",
+"container_samtools" : "/path/to/singularity/image/samtools.sif",
+"container_gatk" : "/path/to/singularity/image/gatk.sif",
+"container_bwa" : "/path/to/singularity/image/bwa.sif",
+"container_snpeff" : "/path/to/singularity/image/snpeff.sif",
+"container_bcftools" : "/path/to/singularity/image/bcftools.sif",
+"filesuffix" : "bam or cram",
+"bucket_dir" : "/path/to/bam or cram/files",
```

Comment:
If this is for mutation calling, they should explicitly name the tumor/normal bams, rather then point at a directory and have the tumor/normal inferred from the name